packages/react-jss/src/withStyles.js

```diff
@@ -80,6 +78,66 @@ export default function withStyles<Theme: {}, S: Styles<Theme>>(
       // $FlowFixMe
       static defaultProps = {...InnerComponent.defaultProps}
 
+      static getSheetClasses(sheet, dynamicRules: ?DynamicRules) {
```

Comment:
We could do this, now?

`if (dynamicRules == null) return sheet.classes`

Reply:
Ohh yeah, if there are no dynamic rules, we can safely return the sheet classes